LICENSE

```diff
@@ -1,14 +1,201 @@
-   Copyright 2013 EvenUp Inc
```

Comment:
You shouldn't actually have to remove this line.  In most cases where we've fixed up license detection before, we've left any copyright claims alone. (Not that we're otherwise removing the original author's copyright, but it just seemed impolite and less likely to cause misunderstandings if we just left what was there alone).